docs/index.rst

```diff
@@ -10,6 +10,7 @@ Other sites that are using this theme:
 
 - Pandas: https://pandas.pydata.org/docs/
 - NumPy: https://numpy.org/doc/stable/
+- Matplotlib: https://matplotlib.org/stable/
 - SciPy: https://scipy.github.io/devdocs/
 - NetworkX: https://networkx.org/documentation/latest/
 - Bokeh: https://docs.bokeh.org/en/latest/
```

Comment:
```suggestion
- Binder: https://docs.mybinder.org/
- Bokeh: https://docs.bokeh.org/en/latest/
- JupyterLab: https://jupyterlab.readthedocs.io/en/latest/
- JupyterHub: https://jupyterhub.readthedocs.io/en/latest/
- Matplotlib: https://matplotlib.org/stable/
- NetworkX: https://networkx.org/documentation/latest/
- NumPy: https://numpy.org/doc/stable/
- Pandas: https://pandas.pydata.org/docs/
- SciPy: https://scipy.github.io/devdocs/
```

While we're at it, I've:

- Sorted alphabetically
- Added two Jupyter docs pages + a Binder page